data/DataHandler.php

```diff
@@ -0,0 +1,247 @@
+<?php
+//==================================
+// Class Definition of DataHandler
+//==================================
+require_once __DIR__.'/../utility/Equipment.php';
+require_once __DIR__.'/../utility/Room.php';
+session_start();
+
+
+class DataHandler {
+    
+    protected const MAX_SPACE = 1000;
+    protected const WAREHOUSE = 'warehouse';
+    protected const NONE = "";
+
+    private string $rooms = 'room_list', $equipments = 'equip_list';
+
+    /**
+     * Constructor for the Handler.
+     * Initializes two arrays containing all equipment and all rooms.
+     * Automatically creates the warehouse.
+     */
+    public function __construct() {
+        $_SESSION[$this->rooms]  ?? $_SESSION[$this->rooms] = [self::WAREHOUSE => new Room(self::WAREHOUSE, self::MAX_SPACE)];
+        $_SESSION[$this->equipments] ?? $_SESSION[$this->equipments] = [];
```

Comment:
Looks to me that $this->rooms and $this->equipment are actually constants and not variables